File: WebAPP/ClientApp/src/dto/parseClass/stayConfig.ts
import { Stay } from "../stay";
import { Activity } from "../activity";
import { Thematic } from "../thematic";
import { Tag } from "../tag";

export class StayConfig {
	public stay: Stay;
	public activities: Array<Activity>;
	public thematics: Array<Thematic>;

	constructor(stay?: Stay, activities?: Array<Activity>, thematics?: Array<Thematic>)

	constructor(stay: Stay, activities: Array<Activity>, thematics: Array<Thematic>) {

		this.stay = stay;
		this.activities = activities;
		this.thematics = thematics;
	}
}

File: WebAPI/DTO/StayConfig.cs
using Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebAPI.DTO
{
    public class StayConfig
    {
        public Stay Stay { get; set; }
        public List<Activity> Activities { get; set; }
        public List<Thematic> Thematics { get; set; }
    }
}
